### PR TITLE
Remove non-serious hubot scripts

### PR DESCRIPTION
**Pull request by hawx**
Pls

### CODE REVIEW

File: external-scripts.json [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
   "hubot-diagnostics",
-  "hubot-google-translate",
   "hubot-hackbot",
   "hubot-heroku-keepalive",
-  "hubot-maps",
-  "hubot-pugme",
   "hubot-redis-brain",
-  "hubot-rules",
-  "hubot-shipit"
+  "hubot-rules"
```
Comment (Codesleuth) [inline external-scripts.json]: Is [`hubot-rules`](https://github.com/hubot-scripts/hubot-rules) a serious script?

Comment (hawx) [reply external-scripts.json]: So true.

**Comment by nick599** [comment]
Why were these removed?

**Comment by hawx** [comment]
They fill the channel with unnecessary noise. 

Don't worry though, I doubt anyone will merge this.

**Comment by andrewseward** [comment]
The whole point of Hackbot is to give people something to talk about, it's ok if it's annoying sometimes

**Pull request closed without merging**